### ISSUE TITLE
Walkers get stuck after pausing the game

### ISSUE THREAD

**Issue by cdsmith-umn** [Keriew/augustus#78]
When I pause the game, various walkers get stuck and no longer move. The sprites' legs are moving, but they're effectively walking in place. This includes prefects, engineers, market ladies, religious workers, sentries and so on. This leads to collapsing buildings and houses running out of food. I have tried simply saving the game and reloading it and this does not fix it. Neither does completely exiting the game. This is in Windows v1.4.1a, though the scenario was originally started in v1.4.0.218 (unstable).

Notably, not all workers get stuck. It seems to be about half of them, which is a rough guess.

**Comment by cdsmith-umn** [comment]
I rolled back to v1.4.0 and this problem does not exist.

**Comment by Keriew** [comment]
Does it always happen when you pause and unpause? I've tried doing that, but it works fine for me. Since reloading doesn't fix that, could you upload the bugged save and augustus.ini file? That could be helpful.

**Comment by cdsmith-umn** [comment]
Sure. I'll get that for you as soon as I can.

**Comment by cdsmith-umn** [comment]
Here it is.
[Lutetia 4x4 palace in progress.zip](https://github.com/Keriew/augustus/files/4766665/Lutetia.4x4.palace.in.progress.zip)

**Comment by Keriew** [comment]
Does that happen every time you pause and unpause, or at least consistently? Or was that a one-off thing?

Could you check if that happens in this build?

[augustus.zip](https://github.com/Keriew/augustus/files/4767699/augustus.zip)

**Comment by cdsmith-umn** [comment]
This was the first time I had seen it happen. I haven't been able to reproduce it since, either. It did not happen using the executable you provided me. I'm not even sure how long it had been happening in the save file I provided to you because I only realized it was an actual problem and not a visual bug when buildings started to collapse and then I observed my labor pool shrinking. I had noticed some walkers "walking in place" several minutes before that all started to happen, but I figured it was a visual bug.

When buildings started to collapse, I couldn't understand why because I was very careful about where engineers and prefects were placed relative to roadblocks and intersections. It was at that point that I clicked on an engineer and the character talked, which surprised me. A couple of minutes after that, my labor pool started to shrink and I went to look at housing to see what was going on and noticed prefects, market ladies, school children, temple workers, and others all walking in place.
